book/02-git-basics/sections/recording-changes.asc

```diff
@@ -184,7 +184,7 @@ M  lib/simplegit.rb
 ----
 
 New files that aren't tracked have a `??` next to them, new files that have been added to the staging area have an `A`, modified files have an `M` and so on.
-There are two columns to the output - the left hand column indicates that the file is staged and the right hand column indicates that it's modified.
+There are two columns to the output - the left hand column indicates the status of the staging area and the right hand column indicates the status of the working tree.
```

Comment:
One more tiny fix: can you correct "left hand" to "left-hand" and "right hand" to "right-hand"?